### ISSUE TITLE
[FEAT] Type Scale Card for Docs

### ISSUE THREAD

**Issue by xelaint** [graycoreio/daffodil#2662]
# :bulb: Feature request

## Feature Name

Type Scale Card for Docs

## The Desired Behavior

The UI and implementation for type scale is written, it just needs to be moved into its own component.

## Your Use Case

As a designer and developer, I would like a design component that is used to display the `@daffodil/design` type scales.

**Comment by damienwebdev** [comment]
@xelaint I think this is a fairly easy issue, can you expand on this issue a little more so that a new contributor could easily pick it up?

**Comment by Ron1005** [comment]
Hey @xelaint and @damienwebdev, I’ve been trying to understand this, but I’m still a bit confused. Could you please explain it more simply?

**Comment by damienwebdev** [comment]
@Ron1005 I believe, but I will let @xelaint clarify exactly where this ultimately needs to go, the Typescale cards are in the [typography file ](https://github.com/graycoreio/daffodil/blob/develop/apps/design-land/src/app/typography/typography.component.html) in our `design-land` app. These components need to be exported to their own separate components and then re-used in the `daffio` app. How exactly we want to do this is a little unclear at the moment, since the component needs to be used in both apps. 

I think a PR for converting these into a shared component exported from `@daffodil/docs` and then used in both `daffio` and `design-land` would be sufficient.

Let me know if you get stuck. You can also join [discord ](https://discord.gg/BdaJVZ53sR) if you need an assist.

**Comment by xelaint** [comment]
@damienwebdev TBH, the type scale card needs to be revisited since this hasn't been looked at since 2021. The current design and docs needs to be updated and should not be built out as is.

**Comment by damienwebdev** [comment]
Ok, @xelaint can you make a figma for it?
